src/wallet/wallet.cpp

```diff
@@ -2535,6 +2534,14 @@ bool CWallet::SelectCoins(const std::vector<COutput>& vAvailableCoins, const CAm
                                    vCoins, setCoinsRet, nValueRet, coin_selection_params, bnb_used)) {
                 return true;
             }
+            // Try with unsafe inputs if they are allowed. This may spend unconfirmed outputs
+            // received from other wallets.
+            if (coin_control.m_include_unsafe_inputs 
```

Comment:
linter appeasement time (sorry about that if it's from my suggestion)
```suggestion
            if (coin_control.m_include_unsafe_inputs
```

Reply:
My bad! Fixed in https://github.com/bitcoin/bitcoin/pull/21359/commits/ce70f322ceb79fe8449d20967c810efcae4881f0